### DISCUSSION TITLE
Error with arbitrary_types_allowed

### DISCUSSION THREAD

**Discussion by kientv** [langchain-ai/langserve#776] [Q&A] [answered]
Please help to solve this problem:

`from langserve import add_routes`
`from langserve.pydantic_v1 import BaseModel`

> pip freeze | grep -E "langchain|langserve|pydantic"
> langchain==0.2.16
> langchain-community==0.2.17
> langchain-core==0.2.41
> langchain-google-genai==1.0.10
> langchain-google-vertexai==1.0.10
> langchain-openai==0.1.25
> langchain-text-splitters==0.2.4
> langserve==0.2.3
> openapi-pydantic==0.4.1
> openapi-schema-pydantic==1.2.4
> pydantic==1.10.9
> pydantic_core==2.23.4
> 

```
lib/python3.11/site-packages/pydantic/v1/validators.py", line 765, in find_validators
    raise RuntimeError(f'no validator found for {type_}, see `arbitrary_types_allowed` in Config')
RuntimeError: no validator found for <class 'langchain_core.messages.base.BaseMessage'>, see `arbitrary_types_allowed` in Config
```

**Comment by eyurtsev** [comment, marked as answer]
Could you include a minimal reproducible example?

`arbitrary_types_allowed` is a pydantic error indicating that some code is attempting to assign attributes that are arbitrary classes (or at least no a pydantic model or a python primitive) to a pydantic base model.

Also consider upgrading to langserve 0.3.x and moving away from the `langserve.pydantic_v1` compatibility bridge

**Reply by kientv** [reply]
Migrated to 0.3.0, thanks :)
